README.md

```diff
@@ -2,49 +2,38 @@
 
 Python project to provide data analysis and insights from website users, using FastAPI.
 
-# Install
 
-This package was coded using Python3.12.7. Check on https://www.python.org/downloads/ how to install Python on your operative system.
-
-It is easier to install all dependencies with poetry.
-
-First, install poetry with
-
-```pipx install poetry```
-(or use `pip` if you want to work in a virtual environment).
-The installation was performed with poetry version 1.8.3, and this or later versions should work. If not, one may install precisely this version using
-
-```pipx install poetry==1.8.3```.
-
-Then, go to the main folder, and install all the dependencies using
-
-```poetry install```.
-
-If using Docker, the version used to test this code is docker 27.2.0.
 
 # Run
 
-To run it locally, you can just type
-
-```fastapi run src/application.py --port 80```
+## docker compose
 
-and then visit `http://0.0.0.0:80` on your browser.
+The easiest and most straightforward way to run this is through docker-compose.
 
-## Docker
+First install Docker. You can find instructions on `www.docker.com`. The version used to test this code is docker 27.2.0.
```

Comment:
and install docker-compose :)
you probably installed docker desktop so you have everything installed right away, but it doesn't work for all os distributions.